src/Symfony/Component/Mailer/Exception/HttpTransportException.php

```diff
@@ -20,14 +20,8 @@ class HttpTransportException extends TransportException
 {
     private $response;
 
-    public function __construct(?string $message, ResponseInterface $response, int $code = 0, \Throwable $previous = null)
+    public function __construct(string $message, ResponseInterface $response, int $code = 0, \Throwable $previous = null)
     {
-        if (null === $message) {
-            trigger_deprecation('symfony/mailer', '5.3', 'Passing null as $message to "%s()" is deprecated, pass an empty string instead.', __METHOD__);
```

Comment:
You may remove `symfony/deprecation-contracts` from `symfony/mailer` as well now.

Reply:
good catch. I forgot we added it 4 days ago